zerver/lib/bugdown/fenced_code.py

```diff
@@ -239,6 +270,28 @@ def done(self):
             output.append('')
         return output
 
+    def format_quote(self, text):
+        # type: (Text) -> Text
+        paragraphs = text.split("\n\n")
+        quoted_paragraphs = []
+        for paragraph in paragraphs:
+            lines = paragraph.split("\n")
+            quoted_paragraphs.append("\n".join("> " + line for line in lines if line != ''))
+        return "\n\n".join(quoted_paragraphs)
```

Comment:
why did you move this?  Might make sense to just have both of these below where `format_quote` was originally so things are nicely alphabetical.